src/Serilog.Extensions.Logging.File/Microsoft/Extensions/Logging/FileLoggerExtensions.cs

```diff
@@ -117,9 +122,10 @@ public static ILoggingBuilder AddFile(this ILoggingBuilder loggingBuilder, IConf
             bool isJson = false,
             long? fileSizeLimitBytes = FileLoggingConfiguration.DefaultFileSizeLimitBytes,
             int? retainedFileCountLimit = FileLoggingConfiguration.DefaultRetainedFileCountLimit,
-            string outputTemplate = FileLoggingConfiguration.DefaultOutputTemplate)
+            string outputTemplate = FileLoggingConfiguration.DefaultOutputTemplate,
+            bool rollOnFileSizeLimit = false)
```

Comment:
Since this is a binary breaking change, we should bump the version number to 4.0.0.

Reply:
changed number